docs/product-business-resource-planner-guide/src/main/asciidoc/AuthoringPlanningAssets/optimizer-domain-editor-con.adoc

```diff
@@ -0,0 +1,43 @@
+[id='optimizer-domain-editor-con']
+= Authoring planning assets with the {PLANNER} domain editor
+:imagesdir: ../..
+
+{PLANNER} leverages the data modeller to create a domain model for constraint satisfaction problems.
+//@doc-link: Domain models link?
+
+In addition to its basic functionality (creating data objects and their properties), you can also use the *Data Objects* editor perspective in {CENTRAL} to enhancing the data model with {PLANNER}-specific data object roles. The `Planning Entity` and `Planning Solution` roles are available in the {PLANNER} dock, accessed by clicking the 
```

Comment:
".... you can also use the *Data Objects* editor perspective in {CENTRAL} to enhancing the data model with ..."

Shouldnt we say "to enhance the data model"? instead of "enhancing the data model"?